themes/hugo-creative-theme/layouts/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="{{ with .Site.LanguageCode }}{{ . }}{{ else }}en-US{{ end }}">
+    <head>
+        {{ partial "head.html" . }}
+    </head>
+    <body id="page-top">
+        {{ if or .Site.Params.navigation.links.about .Site.Params.navigation.links.services .Site.Params.navigation.links.portfolio .Site.Params.navigation.links.contact }}
```

Comment:
The navigation links are also a bit difficult to read / spot in this background. Reverting the opacity from 0.7 to 1.0 (making it plain white) should suffice to fix it. Conversely, the hover color should be changed to use 0.7 opacity instead of being plain white.

Reply:
I have improved this, not quite in the way you suggested. I darkened the background image further, then made the background slightly dark as well. At the suggestion of a friend, I also bumped up the font size. You can see a preview of it here, as before:

https://thanson16.github.io/piqueserver.github.io/

Let me know if you think it is good enough...I personally think it looks good now, but I'm open to changing it more.

Reply:
It's quite readable now, thanks!